openmetadata-docs/content/deployment/docker/volumes.md

```diff
@@ -2,52 +2,115 @@
 title: Docker Volumes
 slug: /deployment/docker/volumes
 ---
+## Advance section
 
-Named volumes can persist data after we restart or remove a container. Also, it’s accessible by other containers.
+Volumes provide the ability to connect specific filesystem paths of the container back to the host machine. If a directory or a file in the container is mounted, changes in that directory  or file can also be seen on the host machine.we are going to use a mapping of a directory present on the host macine with the container path.
 
-For example:
+## Volumes for MYSQL container:
+Following are the changes we have to do while mounting the directory for mysql in OpenMetadata
```

Comment:
Missing a full stop (`.`) in the end of sentence!

Reply:
Done